app.territoiresentransitions.react/src/app/Layout/Header/makeNavItems.ts

```diff
@@ -250,7 +250,7 @@ const filtreItems = (items: TNavItemsList): TNavItemsList =>
     ?.filter((item) => !item.confidentiel)
     .filter((item) => !item.hideToVisitor)
     .map((item) => {
-      return item.hasOwnProperty('items')
+      return Object.prototype.hasOwnProperty.call(item, 'items')
```

Comment:
Pas nécessaire de changer mais pour partager ou échanger.

Dans un cas comme ça, j'ai plutôt tendance à utiliser la syntaxe `'items' in item` que je trouve plus simple à lire. Je crois qu'il y a une petite différence subtile avec ta syntaxe dans le cas de surcharge du prototype de l'objet mais ça semble assez à la marge.

Reply:
Je ne connaissais pas la syntaxe merci ! 👍 

J'ai corrigé car j'avais une erreur sur mon linter. Peut-être peut-on assouplir certaines règles ? Pour le moment c'est celle de next par défaut ? J'ai souvent des erreurs de linter notamment sur les apostrophe par exemple, c'est un peu pénible.

Je vois que t'as fais des changements en ce sens il y a 2 semaines :p
`Object.prototype.hasOwnProperty.call(item, 'to')` dans MenuPrincipal ligne 64

Reply:
Oui les nouvelles règles de linter depuis le passage à Next.js ce sont celles par défaut de Next. Ça reprend globalement les règles standards de eslint + des trucs spécifiques React / Next. Donc on peut assouplir au cas par cas si c'est une règle qui ne nous va vraiment pas, mais j'aurais tendance à quand même suivre ces recommandations le plus possible. Donc plutôt corriger la codebase existante au fur et à mesure, plutôt que d'assouplir les règles.